cfgov/v1/jinja2/v1/home_page.html

```diff
@@ -13,7 +13,25 @@ <h1 class="u-visually-hidden">
 	{{ _('Consumer Financial Protection Bureau') }} {{ _('homepage') }}
 </h1>
 
-{{ carousel.render( carousel_items ) }}
+{% for block in page.content %}
+    {% if block.block_type == 'features' %}
+        <section class="wrapper
+                        wrapper__match-content
+                        block
+                        block__bg
+                        block__border
+                        block__flush-top
+                        u-mb0">
```

Comment:
Would `block__flush-bottom` not work in place of `u-mb0`?

Reply:
I wrote the code he's copying from and am sure I tried that. I think the issue is that it removes the bottom border.

Reply:
Thanks @Scotchester, I was just writing the same. If I replace `u-mb0` with `block__flush-bottom`, the spacing stays the same but we lose the border.